tasks/app/users.py

```diff
@@ -55,32 +47,69 @@ def promote_to_admin(
         print('\nNo updates applied.')
         return
 
-    if user.is_admin:
-        print('The given user is already an administrator:\n\t%r' % (user,))
-        print('\nNo updates applied.')
+    if role == 'Admin':
+        user.set_static_role(User.StaticRoles.ADMIN)
+    elif role == 'Staff':
+        user.set_static_role(User.StaticRoles.STAFF)
+    elif role == 'Internal':
+        user.set_static_role(User.StaticRoles.INTERNAL)
+    elif role == 'Researcher':
+        user.set_static_role(User.StaticRoles.RESEARCHER)
+    elif role == 'Contributor':
+        user.set_static_role(User.StaticRoles.CONTRIBUTOR)
+    elif role == 'Exporter':
+        user.set_static_role(User.StaticRoles.EXPORTER)
```

Comment:
Is this mapping INTERNAL to 'Development Access' and STAFF --> 'Moderator' from the security document?

Reply:
From what @bluemellophone said, Internal is not one of the ones in the security doc and won't be visible outside Houston.
Development access = Staff.